webpack.config.js

```diff
@@ -36,13 +33,14 @@ module.exports =  {
             },
             {
                 test: /\.vue$/,
-                use: {
-                    loader: 'vue-loader',
-                    options: {
-                        esModule: true
-                    }
-                } 
+                use: 'vue-loader'
             }
         ]
-    }
+    },
+    plugins: [
+        new webpack.optimize.CommonsChunkPlugin({
+            name: "vendor",
+            minChunks: Infinity
+        })

```

Comment:
We can spit out any common imports into the vendor chunk as well (could make this a separate common chunk too).

Reply:
Been looking at the way the VS templates do this. They have a seperate config for vendor bits, then use the https://webpack.js.org/plugins/dll-plugin/ to reference this in their main bundle. Which `provide means to split bundles in a way that can drastically improve build time performance.`